src/user/user.spec.js

```diff
@@ -94,9 +94,17 @@ describe("user", () => {
     });
 
     it("fails trying to delete itself", async () => {
-      await expect(
-        xo.call("user.delete", { id: xo._user.id })
-      ).rejects.toMatchSnapshot();
+      const id = await xo.createUser({
+        email: "admin2@admin.net",
+        password: "admin",
+        permission: "admin",
+      });
+      await testWithOtherConnection(
+        { email: "admin2@admin.net", password: "admin" },
+        xo => {
+          expect(xo.call("user.delete", { id })).rejects.toMatchSnapshot();
```

Comment:
You need to await the promise or to return it. Please also update you're branch.